src/components/VmDialog/index.js

```diff
@@ -596,7 +607,14 @@ class VmDialog extends React.Component {
               )}
 
               <dt>
-                <FieldHelp content={msg.bootMenuTooltip()} text={msg.bootMenu()} />
+                {
+                  isUp &&
+                  <React.Fragment>
+                    <span className='pficon pficon-warning-triangle-o' />
+                    &nbsp;
```

Comment:
In this case, use of padding would be better.
Considering time constraint, let's keep it for a follow-up.